test/test_buildkite.py

```diff
@@ -13,15 +13,19 @@
 # limitations under the License.
 # ==============================================================================
 
-import os
 import io
+import os
+import sys
 import unittest
 import warnings
 from shutil import copy
 
+from common import tempdir
 from horovod.runner.common.util import safe_shell_exec
 
-from common import tempdir
+sys.path.append('../.buildkite')
```

Comment:
This is assuming we're running out of the `test` folder.  To make it more generic, we could change it to:

```
sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, '.buildkite'))
```

Reply:
excellent, fixed and decorated with a comment